x-pack/plugins/security_solution/server/lib/telemetry/sender.ts

```diff
@@ -397,7 +397,7 @@ export class TelemetryEventsSender implements ITelemetryEventsSender {
           'X-Elastic-Stack-Version': clusterVersionNumber ? clusterVersionNumber : '8.0.0',
           ...(licenseId ? { 'X-Elastic-License-ID': licenseId } : {}),
         },
-        timeout: 5000,
+        timeout: 10000,
```

Comment:
Agreed with the bump as a remedy, but `Almost all errors we are getting when sending telemetry documents are related to axios timeouts. This PR increments this value.` I disagree with. Upon analyzing the data it seems that this is just client errors when making the request.

Reply:
Do you suggest I change the PR description? otherwise, I'm not following you

Reply:
Yes. For posterity.